tendermint/src/types/time/timestamp.rs

```diff
@@ -336,7 +335,8 @@ impl From<Timestamp> for inner::Timestamp {
     }
 }
 
-impl Protobuf<Timestamp> for Timestamp {}
+impl tendermint_proto::Protobuf<Timestamp> for Timestamp {}
```

Comment:
Do we need both versions?

Reply:
I think we might need the tendermint variant inside our tendermint crate